README.md

```diff
@@ -41,6 +41,8 @@ Available options:
 * `sync_dir`: directory where the files will be synced
 * `skip_file`: any files that match this pattern will be skipped during sync
 * `skip_dir`: any directories that match this pattern will be skipped during sync
+* `use_regex`: use regex for `skip_dir` and `skip_file` options, default `false`
+* `ignore_local_changes`: change to `yes` if you want to ignore local changes, default `no`
```

Comment:
Here you're using `'yes'` and `'no'`, but above you're using `'true'` and `'false'`. Why?